src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.Parser;
+import duke.task.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a Duke chatbot.
+ * A Duke chatbot will save or load the user's tasks in a local file.
+ */
+
+public class Duke {
+
+    private TaskList taskList;
+    private String userName;
+    private String taskFilePath;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Constructor to initialize Duke & its various components.
+     *
+     * @param filePath Path of savefile of tasks information
+     * @return Nothing.
+     */
+    public Duke(String filePath) {
+        taskFilePath = filePath;
+        ui = new Ui();
+        storage = new Storage(taskFilePath, ui);
+        // load TaskList from existing data
+        taskList = new TaskList(storage.loadFileContents());
+    }
+
+    /**
+     * The main method which intializes Duke & runs it.
+     *
+     * @param args Unused.
+     *             return Nothing.
```

Comment:
I dont think that the javadoc needs to state that it returns nothing if it returns nothing